README.md

```diff
@@ -404,37 +403,29 @@ this.alpaca.lastQuote('AAPL').then((response) => {
 ```
 
 ### Websockets
-When to use which websocket?
-1. first of all - if you don't have a funded account you cannot use the
- polygon websocket. <br>The data in the Alpaca websocket is free (currently in
-  beta) and this is your only option.
-2. if you do have a funded account read the docs to understand exactly what
- are the differences between the data streams<br>
- 
-Now since there's is a redundancy in the data we assume that if you use one
- you will not use the other.<br>
-The way you select which websocket to use is by setting the `usePolygon
-` argument when creating the Alpaca instance (see example above). 
+You can use data websocket with or without a funded account. 
 #### Working with websocket
 * The websocket is created when you creating the Alpaca instance
-* `let websocket = alpaca.data_ws`: Get the websocket client instance.
+* `const websocket = alpaca.data_stream_v2`: Get the websocket client instance.
 * `websocket.connect()`: Connect to the Alpaca server using websocket.
-* `client.onConnect(function() {}`: all the following code should be inside
- this function because we should not do anything until we're connected to the
-  websocket.
-* `websocket.subscribe(channels)`: Subscribe to the Alpaca data server and/or
-  the Polygon server.<br>
-  Please note that Polygon and Alpaca servers use different channels. <br>
-    You need to specify the channel you want to
-   subscribe to as specified here:<br>
-    Channels for the Polygon service: `['T.*', 'Q.*', 'A.*', 'AM.*']`.<br>
-    Channels for the Alpaca data service: `['alpacadatav1/T
-    .*', 'alpacadatav1/Q.*', 'alpacadatav1'/AM.*]`
+* `client.onConnect(function() {})`: all the following code will be executed after 
+  the user authentication. You can also subscribe and unsubscribe for data 
+  outside this function.
+* `websocket.subscribeForTrades(["symbol"])`: Subscribe for trades data for the
+  given symbols(s). You can do the same with quotes, bars, dailyBars, statuses
+  and lulds.
+* `websocket.onStockTrade(function(trade) {})`: Get the data and do wathever you 
+  want with it inside this function.
+* `websocket.unsubscribeFromTrades(["symbol"])`: Unsunscribe from symbol(s).
+* `websocket.onDisconnect(function() {})` and `websocket.disconnect()`: the function
+  inside the onDisconnect will run when you cann disconnect, then closes the connection
+  between the client and server.
+<br><br>
```

Comment:
I think these wont create breaklines, maybe this: `<br/><br/>`

Reply:
It works